src/main/java/com/ovcharenko/carrental/controller/CarServiceController.java

```diff
@@ -0,0 +1,42 @@
+package com.ovcharenko.carrental.controller;
+
+import com.ovcharenko.carrental.model.Car;
+import com.ovcharenko.carrental.services.CarService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+public class CarServiceController {
+    @Autowired
+    CarService carService;
+
+    @RequestMapping(value = "/cars")
+    public ResponseEntity<Object> getCars() {
+        return new ResponseEntity<>(carService.findAll(), HttpStatus.OK);
+    }
+
+    @RequestMapping(value = "/cars/{id}", method = RequestMethod.GET)
```

Comment:
You can use @GetMapping/ @PostMapping instead of @RequestMapping
and also according to the REST specification in the plural name of the endpoints only when doing getAll